mu-plugins/blocks/time/index.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Format Name: Time
+ * Description: Attempts to parse a time string like <code>Tuesday, April 5th 2022, at 15:00 UTC</code> and creates a format that shows it in the viewers local time zone.
+ *
+ * @package wporg
+ */
+
+namespace WordPressdotorg\MU_Plugins\Time;
+
+add_action( 'init', __NAMESPACE__ . '\init' );
+
+/**
+ * Enqueues the scripts for the Time format in the block editor and frontend using the metadata loaded from the `block.json` file.
+ */
+function init() {
+	// Add the JS script to add the Time formatting option to the toolbar. The dependencies are autogenerated in block.json,
+	// and can be read with `wp_json_file_decode` & `register_block_script_handle.
+	$metadata_file = __DIR__ . '/build/block.json';
+	$metadata = wp_json_file_decode( $metadata_file, array( 'associative' => true ) );
+	$metadata['file'] = $metadata_file;
+
+	$editor_script_handle = register_block_script_handle( $metadata, 'editorScript', 0 );
+	add_action(
+		'enqueue_block_assets',
+		function() use ( $editor_script_handle ) {
+			if ( wp_should_load_block_editor_scripts_and_styles() && is_admin() ) {
+				wp_enqueue_script( $editor_script_handle );
+			}
+		}
+	);
+
+	$view_script_handle = register_block_script_handle( $metadata, 'viewScript', 0 );
+	add_action(
+		'wp_enqueue_scripts',
+		function() use ( $view_script_handle ) {
+			if ( ! is_admin() ) {
+				wp_enqueue_script( $view_script_handle, null, null, filemtime( __DIR__ . '/build/convert_times.js' ), true );
```

Comment:
It's generally best to use a literal version number instead of `filemtime()` in production, otherwise every page view requires a FS read. We're not exactly consistent about that across the codebase though. Not a blocker.

Reply:
This script is already registered by the `register_block_script_handle` function with the generated dependencies & version string, so you can enqueue it by just using the handle. To also output it in the footer, you can change the `group` — that's all the `$in_footer` [parameter does](https://github.com/WordPress/wordpress-develop/blob/bea4307daa21a3f97d159898b0ac676332e0e7de/src/wp-includes/functions.wp-scripts.php#L173-L184).

```suggestion
				wp_enqueue_script( $view_script_handle );
				wp_script_add_data( $view_script_handle, 'group', 1 );
```